libindy/src/services/anoncreds/helpers.rs

```diff
@@ -165,7 +165,7 @@ mod tests {
     #[test]
     fn transform_u32_to_array_of_u8_works() {
         let int = 1958376517;
-        let answer = vec![0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 116, 186, 116, 69];
+        let answer = vec![116, 186, 116, 69];
```

Comment:
Not obvious format of ints: should be in hex format.
Also `transform_u32_to_array_of_u8` should have comment or suffix about byte order